validation/src/test/resources/application.conf

```diff
@@ -15,5 +15,6 @@ akka {
 
 hmda {
   isDemo = false
+  edits.demoMode = false
```

Comment:
Why is this config being added to the validation and persistence project?

Reply:
It's useful to me when I'm developing on a branch. Sometimes in that process I'll go into a specific project in sbt (like `project persistence`) and run the specs there. When I did that recently, I found out that I needed this config for the specs to pass. 

(Note that both places where I added `edits.demoMode = false` are in `test` config files, not `main`)